api_reflector/admin.py

```diff
@@ -75,6 +75,10 @@ class EndpointView(RestrictedView):
 
     form_excluded_columns = ("responses",)
     form_widget_args = {"responses": {"disabled": True}}
+    column_searchable_list = (
```

Comment:
For whatever reason, the search box looks weird with the current `darkly` theme

With darkly:
<img width="595" alt="image" src="https://user-images.githubusercontent.com/53666594/194266257-f1e1d4b8-8ce9-416c-acfa-1aa84db5d4ee.png">

With [slate](https://bootswatch.com/slate/) :
<img width="613" alt="image" src="https://user-images.githubusercontent.com/53666594/194266199-4db433fd-e2ad-4804-a307-9fc0d8b80dda.png">

But for now, Im happy to remove this change out if it looks too weird